app/models/admin_menu.rb

```diff
@@ -0,0 +1,76 @@
+class AdminMenu
+  # On second level navigation with more children, we reference the default tabs controller. i.e look at developer_tools
+  # rubocop:disable Metrics/BlockLength
+  ITEMS = Menu.define do
+    scope :people, [
+      item(name: "people", controller: "users"),
+    ]
+
+    scope :content_manager, [
+      item(name: "posts", controller: "articles"),
+      item(name: "badges", children: [
+             item(name: "badge library", controller: "badges"),
+             item(name: "badge achievements"),
+           ]),
+      item(name: "organizations"),
+      item(name: "podcasts"),
+      item(name: "tags"),
+    ]
+
+    scope :customization, [
+      item(name: "config"),
+      item(name: "HTML variants", controller: "html_variants"),
```

Comment:
`HTML`'s casing apparently isn't respected in the final UI, so think it can be kept lower cased like everything else:

![Screenshot_2021-03-05 Html Variants](https://user-images.githubusercontent.com/146201/110117587-5a292600-7db9-11eb-9f29-48c87d31ceed.png)

Reply:
✅